services/gateway/apis/http-version3/swagger/definitions/blocks.json

```
@@ -100,6 +100,21 @@
                     }
                 }
             },
+            "totalBurnt": {
+                "type": "string",
+                "example": "0",
+                "description": "Total amount of LSK that have been burnt in this Block"
+            },
+            "networkFee": {
+                "type": "string",
+                "example": "15000000",
+                "description": "The total amount of fees associated with the block."
+            },
+            "totalForged": {
+                "type": "string",
+                "example": "65000000",
+                "description": "Total amount of LSK that have been forged in this Block.\nConsists of fees and the reward.\n"
```

The `reward` property seems to be missing in the block definition.
```suggestion
                "description": "The total amount of Lisk (in Beddows) that was burnt in this block. It is the sum total of minimum fees for each transaction contained within the block."
            },
            "networkFee": {
                "type": "string",
                "example": "15000000",
                "description": "The total amount of Lisk (in Beddows) that the block generator makes from the transaction fees after burning the minimum fees."
            },
            "totalForged": {
                "type": "string",
                "example": "65000000",
                "description": "Total amount of Lisk (in Beddows) that has been forged in this block. It is a sum of all the transaction fees and the block reward."
```